index.html

```diff
@@ -0,0 +1 @@
+<html>
```

Comment:
Really not sure about that one.
How about : 
```suggestion
<html>
</html>
```